lib/helpers/consts.js

```diff
@@ -37,6 +37,7 @@ const CALLBACK_PROPERTIES = [
   'state',
   'token_type',
   'session_state',
+  'resource',
```

Comment:
`resource` is not a IANA registered authorization response property, please remove this.

Reply:
Sorry, this wasn't suppose to be part of this pull request. Would you accept a separate pull request for this to support resource indicators https://tools.ietf.org/html/draft-ietf-oauth-resource-indicators-01 ?

Reply:
just callback wouldn't be full support. Let me take a note and include this in the 3.x due in a month, i'll do a full support there.

Reply:
Sounds good. Do you still need the pull request for extra claims in JWT client assertion? If so I'll branch the change and re-upload a pull request.

Reply:
no, you can close this one. It's on my list to do something about it.